tensorflow/tools/pip_package/build_pip_package.sh

```diff
@@ -176,6 +176,18 @@ function build_wheel() {
   rm -f MANIFEST
   echo $(date) : "=== Building wheel"
   "${PYTHON_BIN_PATH:-python}" setup.py bdist_wheel ${PKG_NAME_FLAG} >/dev/null
+
+  # Fix include path by moving tensorflow_core/include/tensorflow{_core,}
+  WHEEL_TMPDIR=$(mktemp -d)
+  WHEEL=$(readlink -f dist/*.whl)
+  unzip $WHEEL -d $WHEEL_TMPDIR > /dev/null
+  rm $WHEEL
+  pushd $WHEEL_TMPDIR > /dev/null
+  mv tensorflow_core/include/tensorflow{_core,}
```

Comment:
I think we only need this line but immediately after line 172 above:

```bash
mv tensorflow tensorflow_core   # this is line 172
mv tensorflow_core/include/tensorflow{_core,}   # this is the line
```

This way we don't unzip and rezip the wheel and potentially breaking the contract in manifest.

Reply:
I'm going to approve as is for now and fix that tomorrow after we have new pips

Reply:
That’s weird, why do we have them under tensorflow_core in the first place, before bdist_wheel? I’ll take a closer look at your suggested fix.

Reply:
> This way we don't unzip and rezip the wheel and potentially breaking the contract in manifest.

The MANIFEST.in file seems long broken anyway. Do we need to fix them as well?